BangazonAPI/BangazonAPI/Models/Department.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BangazonAPI.Models
+{
+    public class Department
+    {
+        public int Id { get; set; }
+
+        public int Budget { get; set; }
+
+        public string Name { get; set; }
+    }
+}
```

Comment:
Ticket #7 says that "If the query string parameter of ?_include=employees is provided, then all employees in the department(s) should be included in the response."  Does the Model allow for this?

Reply:
I added a List of <Employee> to the Department Model